.github/workflows/suite-native_ci.yml

```
@@ -0,0 +1,50 @@
+name: suite-native develop ci
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - "packages/suite-*/**"
+      - "packages/icons/**"
+  workflow_dispatch:
+
+jobs:
+  android_develop:
+    defaults:
+      run:
+        working-directory: packages/suite-native
+    runs-on: ubuntu-latest
+    environment: develop
+    timeout-minutes: 45
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup react-native kernel and increase watchers
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+      - name: Decode files
+        env:
+          ENCODED_STRING_KEYSTORE: ${{ secrets.DEVELOP_KEYSTORE_FILE }}
+          ENCODED_STRING_JSON_FILE: ${{secrets.GOOGLE_PLAY_JSON_KEY}}
+        run: |
+          TMP_FILE_PATH="${RUNNER_TEMP}/app"
+          mkdir "${TMP_FILE_PATH}"
+          echo $ENCODED_STRING_KEYSTORE | base64 -di > "${TMP_FILE_PATH}/release.keystore"
+          edho $ENCODED_STRING_JSON_FILE > "${TMP_FILE_PATH}/firebase_key.json"
```

```suggestion
          echo $ENCODED_STRING_JSON_FILE > "${TMP_FILE_PATH}/firebase_key.json"
```
Sorry about nitpicking in draft. I'll just keep you from being sad after the first try :).

---

Thnx 😆